src/mousetracker.js

```diff
@@ -318,6 +318,25 @@
         },
 
         /**
+         * Returns {Array.<OpenSeadragon.MouseTracker>} excluding the given pointer device type
```

Comment:
Sorry for not noticing this before, but this doc comment should be improved...something like:

"Returns the {@link OpenSeadragon.MouseTracker.GesturePointList|GesturePointList} for all but the given pointer device type"